integration/up/autocreate_test.go

```diff
@@ -85,6 +85,23 @@ dev:
     forward:
     - 8012:8080
     autocreate: true
+`
+
+	autocreateManifestV2WithDefinedUser = `
+name: test
+image: okteto.dev/test:latest
+command: echo done
+autocreate: true
+persistentVolume:
+  enabled: false
+sync:
+- .:/app
+`
+
+	dockerfileDefinedUser = `
+FROM alpine
+USER 1001
+COPY --chown=1001:1001 . /app
```

Comment:
There is some magic in the test. With build v1, if the image is not present, the image is automatically built. Would it make sense to change to use a manifest v2? In that case, the manifest would have the build section and would make this much easier to understand.

Anyways, really good test to catch it and leave it like that if you want or if you think it would take much more time. Just one thing, if we leave it with manifest v1, change the name of the variable `autocreateManifestV2WithDefinedUser` to `autocreateManifestV1WithDefinedUser`

Reply:
Yes, you are right. I have added the same test for v2 of the manifest. I keep the test for v1 for proximity to the scenario that has caused us problems before

Reply:
I didn't want you to build the test for the 2 versions but yeah, agree it is better! Thanks!!